packages/generator/src/internal.ts

```diff
@@ -0,0 +1,28 @@
+export * from './action-function-import';
+export * from './aggregator-package';
+export * from './batch';
+export * from './complex-type';
+export * from './edmx-parser';
+export * from './edmx-to-vdm';
+export * from './entity';
+export * from './enum-type';
+export * from './request-builder';
+export * from './sdk-metadata';
+export * from './service';
+export * from './swagger-parser';
+export * from './file-generator';
+export * from './generator-options';
+export * from './generator-utils';
+export * from './generator';
+export * from './global-name-formatter';
+export * from './imports';
+export * from './input-path-provider';
+export * from './internal-prefix';
+export * from './name-formatting-strategies';
+export * from './package-description';
+export * from './reserved-words';
+export * from './service-generator';
+export * from './service-mapping';
+export * from './service-name-formatter';
+export * from './typedoc';
+export * from './vdm-types';
```

Comment:
Same as the openapi generator. I would also add `generator-cli.ts` to internal so it contains all ts files.

Reply:
@jjtang1985 - I had added it initially in both packages, but it breaks the e2e tests. I debugged this and found that both  [cli.ts](https://github.com/SAP/cloud-sdk-js/blob/f4ad2302c7126347bc2124a016278d84f54dbbba/packages/openapi-generator/src/cli.ts#L12) and [generator-cli.ts](https://github.com/SAP/cloud-sdk-js/blob/f4ad2302c7126347bc2124a016278d84f54dbbba/packages/generator/src/generator-cli.ts#L17) directly calls the parse command line args methods. Therefore when internal.ts file loads all the modules, when it loads these files, it ends up calling the parse methods and fails. Thats why I have left them out.